modules/apps/foundation/frontend-theme/responsive-boat-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,285 @@
+$mobileNavExpandTime: 0.4s;
+$navBtnHeight: 40px;
+$navBtnPadding: 20px;
+$navLinkFadeTime: 0.5s;
+$myYellow: #FFDD00;
+
+/* All styling is nested within #wrapper selector to avoid accidentally styling the liferay menus */
+
+#wrapper {
+	background-attachment: fixed;
+	background-color: #A9A9A9;
+	background-image: url("../images/background.jpg");
+	background-position-x: center;
+	background-size: cover;
+	padding: 0;
+	text-align: center;
+
+	#banner {
+		background: rgba(0, 0, 0, 0.5);
+		color: #FFF;
+		padding-bottom: 500px;			// add space below navbar to copy the Wordpress theme
```

Comment:
I noticed your spacing for the inline CSS comments are all different.  We should keep them all consistent throughout the file.